widget/radio.go

```diff
@@ -20,6 +20,20 @@ type radioRenderer struct {
 	radio   *Radio
 }
 
+func removeDuplicatedOptions(options []string) []string {
```

Comment:
I am not sure about the naming here. This function takes in, and returns, a slice of strings - it does not really know what "options" are...
Perhaps it should be a function on the radio type? (and then it would not need the input/output variables)?
Of perhaps it should have a name that represents it's generic nature such as "removeDuplicateStrings". 
This function is currently available to the whole widget package - this may be a good thing as perhaps CheckBox should have the same filter?

I will leave the solution up to you - or chat on Slack :)